packages/web-app-files/tests/components/BatchActions.spec.js

```diff
@@ -0,0 +1,250 @@
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import DesignSystem from 'owncloud-design-system'
+import Vuex from 'vuex'
+import getters from '../../src/store/getters'
+import stubs from '../../../../tests/unit/stubs'
+import BatchActions from '../../src/components/BatchActions.vue'
+
+const localVue = createLocalVue()
+localVue.use(DesignSystem)
```

Comment:
Is it really necessary to mount ODS here? In unit tests, I'd prefer to have stubs instead...